core-bundle/src/Command/InstallCommand.php

```diff
@@ -60,6 +60,11 @@ class InstallCommand extends Command
      */
     private $webDir;
 
+    /**
+     * @var array
```

Comment:
```suggestion
     * @var array<BundleInterface>
```
Not sure though if all bundles implement this interface? otherwise `array<object>`?

Reply:
Well, it should be `array<string, string>`, if anything (see below).

Reply:
Yes, `array<string, string>` should it be then I think. This should limit the confusion about what this array is :)